src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -314,22 +317,27 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
 
     val stderrFileLength = Try(Files.size(stderr)).getOrElse(0L)
     val failOnErrFlag = failOnStderr.getOrElse("false").toBoolean
-    val rc = returnCode.contentAsString.stripLineEnd.toInt
+    lazy val rc = Try(returnCode.contentAsString.stripLineEnd.toInt)
 
     if (processReturnCode != 0) {
       TaskFinalStatus(Status.Failed, FailureTaskResult(
         new IllegalStateException("Execution process failed."), processReturnCode, stderr.toString))
+    } else if (rc.isFailure) {
+      // case where docker fails.
+      TaskFinalStatus(Status.Failed, FailureTaskResult(
+        new IllegalStateException("Could not get RC file."), processReturnCode, stderr.toString))
```

Comment:
I commented on an older version of this and it disappeared from view, apologies if you see this twice. :smile: 

The exception associated with the `rc.isFailure` case should be given as the cause for this `IllegalStateException`.